src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Grid, IconButton} from '@material-ui/core';
+import { makeStyles} from '@material-ui/core/styles';
+import { Facebook, Twitter, Instagram, LinkedIn } from '@material-ui/icons';
+
+function Footer() {
+	const useStyles = makeStyles(theme => ({
+		footer: {
+			position: 'fixed',
+			left: 0,
+			bottom: 0,
+			width: '100%',
+			textAlign: 'center',
+			borderTopStyle: 'ridge',
+		}
+	}));
+
+	const classes = useStyles();
+	return (
+		<footer className={classes.footer}>
+			<Grid container spacing={1}>
+				<Grid item xs={12}>
+					<Grid item xs={12}>
```

Comment:
Hvorfor er et grid i et grid i et grid nødvendigt? Og hvorfor kun xs? Skal det ikke defines for alle størrelser, eller er det specielt for xs size skærme?
(faktisk spørgsmål)

Reply:
til det forrige så var det mig der havde en plan, men endte med at lave noget andet og så har jeg bare glemt det, så det fikser jeg lige. 
tilæ det senere. I alle eksempler som jeg har set er det det de gør hvis de ikke definerer special cases. Så jeg tror at det bare fungerer for alt der er >= xs

Reply:
ah, nice - det er lækkert, i bootstrap mener jeg at man bør definere alle størrelser, og det er lidt trash